README.md

```diff
@@ -77,24 +96,28 @@ The code for the script you just ran is `index.js`.
 
 Let's go through the skeleton part by part to understand what's going on:
 
-    const daml = require('@da/daml-ledger');
+    const ledger = require('@digital-asset/daml-ledger');
+
+    const daml = ledger.daml;
 
     const uuidv4 = require('uuid/v4');
 
 The first line loads the bindings and allows you to refer to them through the `daml` object.
```

Comment:
The object is now called `ledger`.

Reply:
Addressed by https://github.com/digital-asset/ex-tutorial-nodejs/pull/7/commits/f20379f70347c217327b508d449245e34a8b9801